### PR TITLE
Feature/big reveal

### PR DESCRIPTION
**Pull request by tompsherman**
# Description

Fixes #

- What work was done?
- big reveal animaiton sequence built and put in place
- Why was this work done?
- a behest of stakeholders
- What feature / user story is it for?
- [FT 2] As a kid user, I want to be able to see the weekly results in a fun way on Friday
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/Animations/FaceoffReveal.js [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+  // dynamicBackgroundColor will be determined by which matchup is being animated--
+  // we may need to create a variable / key-value pair to track which faceoff is occurring
+  let backColorArray = ['#438eac', '#ffde3b', '#e97451', '#C9E952'];
+  let dynamicBackgroundColor = backColorArray[Math.floor(Math.random() * 4)];
+  //    hex codes:
+  //      4. boston-blue -- #438eac
+  //      3. bright-sun -- #ffde3b
+  //      2. burnt-sienna -- #e97451
+  //      1. conifer -- #C9E952
+  console.log('matchup type', matchupType);
```
Comment (kubes2020) [inline src/components/pages/Animations/FaceoffReveal.js:52]: Since it's still a work in progress I think we can leave this console.log

**Review by kubes2020 — APPROVED** [review]
Looking good Tom! I couldn't have done it better myself!

**Review by LizDrumm — APPROVED** [review]
Can't wait to see these animations in action!

File: src/components/pages/Animations/FaceoffReveal.js (hunk 1 above)
Comment (AprilAshby) [reply src/components/pages/Animations/FaceoffReveal.js:52]: I agree. They should be cleaned up eventually, but these can be useful for the next cohort coming in. I know I used several leftover console logs from past teams to help me figure out where to even begin looking.

File: src/components/pages/MatchUp/FaceoffContent.js [hunk 2]
```diff
@@ -34,11 +47,17 @@ const FaceoffContent = props => {
...
+        <FaceoffReveal
+          animationDynamicContent={props.content}
+          setToggle={setToggle}
+        />
+      ) : null}
     </div>
   );
 };
 
-//ERRLOG CURRENT:  what voodoo is this subcomponent within a component?
```
Comment (AprilAshby) [inline src/components/pages/MatchUp/FaceoffContent.js:41]: Thanks for the clean up - I've been meaning to get around to it for a while now

**Review by AprilAshby — APPROVED** [review]
This is an insane amount of work @tompsherman and @kubes2020. I know the stakeholders were loving the animation. I hope I get to see it with the actual avatars. Guess this sort of thing is what keeps Story Squad teams coming back.

**Pull request merged**